templates/santa/message.html.twig

```diff
@@ -4,182 +4,204 @@
-                    This is usually the right place to give more details like a <strong>deadline</strong>
-                    or a <strong>price limit</strong>!
-                </p>
-
-                <textarea name="message" id="message" rows="5">{{ message }}</textarea>
-            </div>
-
-            <div class="user-field message-step">
-                <h3>Add optional details about the participants</h3>
-
-                <p>
-                    If you want, you can add some <strong>details about the participants</strong>.
-                    Those details will be <strong>sent only to the Secret Santa of each user</strong>.
-                    You can use those notes to inform of <strong>address of delivery</strong>,
-                    <strong>food allergy</strong>, or <strong>whatever information</strong>
-                    participants need to know about their target.
-                </p>
-
-                {% apply spaceless %}
-                    <div class="user-container">
-                        <div class="user-list">
-                            {% set key = 0 %}
-                            {% set input_placeholders = {
-                                0: 'An address ;',
-                                1: 'A food allergy ;',
-                                2: 'Or whatever you want ...',
-                            } %}
-                            {% for user in availableUsers %}
-                                {% if user.identifier in selectedUsers %}
-                                    <label class="user-item notes" for="note-{{ user.identifier }}">
-                                        <span class="user-column">
-                                            <span class="user-data">
-                                                {% block user_item %}{% endblock %}
-                                            </span>
-                                            <span class="note-length">
-                                                <span data-length-field="note-{{ user.identifier }}">{{ notes[user.identifier]|default('')|length }}</span>&nbsp;/&nbsp;400 characters
-                                            </span>
-                                        </span>
-                                        <input id="note-{{ user.identifier }}" name="notes[{{ user.identifier }}]" value="{{ notes[user.identifier]|default('') }}" type="text" {% if key in input_placeholders|keys %}placeholder="{{ input_placeholders[key] }}"{% endif %} />
-                                    </label>
-                                    {% set key = key + 1 %}
+        {{ form_start(form, {
+            'attr': {'id': 'run_form'},
+            'method': 'POST',
+            'action': path('message', { application: application })
+        }) }}
+        {# Prevent implicit submission of the form - no need for JS here (see https://stackoverflow.com/a/51507806/1917092) #}
```

Comment:
Could you keep the indentation between form_start & form_end to make diff smaller?